components/eamxx/src/physics/shoc/shoc_functions.hpp

```diff
@@ -69,6 +69,16 @@ struct Functions
   using WorkspaceMgr = typename ekat::WorkspaceManager<Spack,  Device>;
   using Workspace    = typename WorkspaceMgr::Workspace;
 
+  // This struct stores runtime options for shoc_main
+ struct SHOCRuntime {
+   SHOCRuntime() = default;
+   // Runtime options for isotropic_ts
+   Scalar lambda_low    = 0.001;
```

Comment:
Let's avoid putting defaults in several places in the source code, especially for configurable parameters. Let's just put the defaults in the XML file.

Reply:
I've made the change in the latest version.  I had added the defaults here for the sake of unit tests, so that users wouldn't have to know the order of the parameters when initializing a SHOCRuntime struct.  But presumably we won't be adding new runtime options often so probably not necessary.